preamble.tex

```diff
@@ -101,6 +101,10 @@
 \end{nonnormative}
 }
 
+% Semantical markup with formatting
+\newcommand{\firstuse}[1]{\emph{#1}}
+\newcommand{\filename}[1]{\textit{#1}}
```

Comment:
Hardcoding something to `\textit`  has the danger that that is not aware of the font around it. 
Example:
```
\emph{Make sure to save the result to \filename{result.out}.}
```
Here the filename will no longer stand out. I suggest to use 
`\newcommand{\filename}[1]{\texttt{#1}}` instead.

Reply:
Right it wouldn't stand out.  I just happen to have a personal preference for not using fixed-width font for filenames, as I like to reserve fixed-width fonts for code.

For me, going with `\emph` would be a better way to make it stand out also inside `\emph`.

Let's see if there are others with an opinion…

Reply:
Alternatively:
```
\newcommand{\filename}[1]{\mbox{#1}}
```

Reply:
The `\fbox` is too disruptive in my opinion.

Reply:
I can see that. San-serif might be an alternative then.

Reply:
```
\newcommand{\filename}[1]{\textisf#1}}
```

Reply:
I guess you meant `\textsf`.  Pushing that for now.